### PR TITLE
Add more QMargins/QMarginsF autotests

### PR DESCRIPTION
**Pull request by Montel**

### CODE REVIEW

File: tests/qt_types_standalone/rust/src/qmargins.rs [hunk 1]
```diff
@@ -30,3 +32,11 @@ fn read_qmargins(m: &QMargins) -> bool {
 fn clone_qmargins(m: &QMargins) -> QMargins {
     m.clone()
 }
+
+fn test_is_null(m: &QMargins) -> bool {
+    m.is_null()
+}
+
+fn add_margins(m1: QMargins, m2: QMargins) -> QMargins {
+   m1 + m2
```
Comment (ahayzen-kdab) [inline tests/qt_types_standalone/rust/src/qmargins.rs:41]: `cargo fmt` is failing here

File: tests/qt_types_standalone/rust/src/qmarginsf.rs [hunk 2]
```diff
@@ -30,3 +32,11 @@ fn read_qmarginsf(m: &QMarginsF) -> bool {
 fn clone_qmarginsf(m: &QMarginsF) -> QMarginsF {
     m.clone()
 }
+
+fn test_is_nullf(m: &QMarginsF) -> bool {
+    m.is_null()
+}
+
+fn add_marginsf(m1: QMarginsF, m2: QMarginsF) -> QMarginsF {
+   m1 + m2
```
Comment (ahayzen-kdab) [inline tests/qt_types_standalone/rust/src/qmarginsf.rs:41]: `cargo fmt` is failing here

**Review by ahayzen-kdab — APPROVED** [review]
LGTM :-)

**Review by ahayzen-kdab — DISMISSED** [review]

**Review by ahayzen-kdab — APPROVED** [review]

**Review by LeonMatthesKDAB — APPROVED** [review]
LGTM

**Pull request merged**